rmf_task/src/rmf_task/BinaryPriorityScheme.cpp

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#include <rmf_task/BinaryPriorityScheme.hpp>
+
+#include "BinaryPriority.hpp"
+#include "BinaryPriorityCostCalculator.hpp"
+
+namespace rmf_task {
+
+//==============================================================================
+std::shared_ptr<Priority> BinaryPriorityScheme::make_low_priority()
+{
+  return nullptr;
```

Comment:
I feel a bit conflicted about this.

On one hand, it makes sense to use `nullptr` since it would be more efficient than allocating an object to the heap and dynamically casting it to view the value.

On the other hand, it could be confusing for users to receive a `nullptr` when they've asked for an object to be created. It might give the impression that something failed inside the `rmf_task` library.

Let's add documentation to the `make_low_priority` function to explain that the current implementation of this function simply returns a `nullptr`, and that's the normal, expected behavior until further notice.

Reply:
Have added documentation https://github.com/osrf/rmf_core/pull/278/commits/68ec61f4cdee5eae420a912d29b750b00acb1826